yolort/utils/visualizer.py

```diff
@@ -4,16 +4,16 @@
 
 import numpy as np
 import torch
+import yolort.utils.dependency as _dependency
 from PIL import Image
 from torch import Tensor
 from yolort.v5.utils.plots import Colors
 
-try:
+if _dependency.is_module_available("cv2"):
     import cv2
-except ImportError:
-    cv2 = None
 
 
+@_dependency.requires_module("cv2")
```

Comment:
This is not important, thought we suggest to move the decorator `@_dependency.requires_module("cv2")` before the following `self.draw_box()` and `self.draw_text()` method

https://github.com/zhiqwang/yolov5-rt-stack/blob/bf05b4b510b2a41ce4d8975b9b1be33784d29450/yolort/utils/visualizer.py#L175
https://github.com/zhiqwang/yolov5-rt-stack/blob/bf05b4b510b2a41ce4d8975b9b1be33784d29450/yolort/utils/visualizer.py#L196